conf/docker-aio/entrypoint.bash

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
-
+export LANG=en_US.UTF-8
```

Comment:
What's this LANG environment variable for? Should we add a comment?

Reply:
Being defensive for cases where the host system is using a non-US, non-UTF-8 locale.